### ISSUE TITLE
Unable to view new FAQ on front end - v2.9.0-alpha3

### ISSUE THREAD

**Issue by mtompkins** [thorsten/phpMyFAQ#1042]
Spun up a new install - default settings (basic, no groups).
Entered a new FAQ in the backend, entered as MD using editor.
Checked preview - looked good (change to highlighter seems to have resolved the apostraphe issue)

Tried to view the FAQ in front end, no HTML returned...no errors shown in apache. Sounds like possible additional authorization bugs at a guess.  Should be easy enough to reproduce.

**Comment by thorsten** [comment]
Tested it and works for me.

**Comment by thorsten** [comment]
DEBUG mode enabled?

**Comment by mtompkins** [comment]
Both settings. 
Will spin up a new instance and confirm when time allows.

**Comment by thorsten** [comment]
Okay, thanks!

**Comment by thorsten** [comment]
Wait for this issue before releasing the third alpha!

**Comment by mtompkins** [comment]
Confirmed multiple issues:
Spun up a new install with current dev branch (35ef28789e). Install went well except for need to manually create folders.

Config wizard went fine.

Configured to use MD editor as this is my editor of choice.
Entered a FAQ with the following code which identified previous bug:
```
for i in `sudo docker images|grep \<none\>|awk '{print $3}'`;do sudo docker rmi $i;done
```
After save, this code becomes altered and no longer valid:
```
for i in `sudo docker images|grep \|awk &#39;{print $3}&#39;`;do sudo docker rmi $i;done
```
Lastly, I confirm that the front end does not render the faq.

I won't be able to provide additional information as I will be removing the test VM, however the above should be easily to reproduce.

**Comment by thorsten** [comment]
Thanks a lot, will try to fix that before I release the third alpha!

**Comment by thorsten** [comment]
Fixed the unvalid output for your docker example. What do you mean by "Lastly, I confirm that the front end does not render the faq."

It renders for me correctly. Please provide a screenshot to clarify your issue. Thanks!

**Comment by mtompkins** [comment]
Unfortunately I cannot provide a screenshot. All I can give you is no FAQ was produced on the front end in either FF or IE...same condition as originally reported.

**Comment by thorsten** [comment]
Please try to enable the debug log in inc/Bootstrap.php to raise a visible error. I think there's a fatal error because the Markdown library is missing on your system.

**Comment by mtompkins** [comment]
Just a thought, maybe it is PHP version related. It was tested on PHP Version 5.5.26 which is a bit dated.

**Comment by thorsten** [comment]
I use 5.5.x, too

**Comment by mtompkins** [comment]
OK, was hoping for something easy. When I can I'll try and provide the requested info but as per the original request earlier in the thread there were no errors produced last time we checked. 

As per markdown being missing, if that were the case the preview pane on the backend wouldn't render properly - which it does (at a guess).

**Comment by thorsten** [comment]
The markdown class is added different in the front end compared to the backend.

**Comment by thorsten** [comment]
@mtompkins Did you tried the third alpha version?

**Comment by mtompkins** [comment]
Yes, same condition. Used both your compiled package and my own.

**Comment by thorsten** [comment]
Strange, please enable the debug mode in inc/Bootstrap.php and check if you see any errors. That's really strange!

**Comment by theonlyringo** [comment]
Not completely sure if it is related, but I am having the same issues with the faqs not displaying. After enabling debugging this is what shows. 
![screencapture-10-1-1-142-new-index-php-1441750811395](https://cloud.githubusercontent.com/assets/5071472/9748934/0e388490-564e-11e5-8635-cb81c718d73c.png)

**Comment by mtompkins** [comment]
Posting this in the event it helps find a common cause:
mysql  Ver 15.1 Distrib 5.5.43-MariaDB, for Linux () using readline 5.1

**Comment by thorsten** [comment]
Yes, that's the issue, the CREATE TABLE queries are wrong. Will fix that asap. Thanks!

**Comment by thorsten** [comment]
@mtompkins Please try this query:

    ALTER TABLE faqdata ENGINE = MyISAM;

... and try it again. Thanks!

**Comment by mtompkins** [comment]
I'll have to spin up a new instance. Please let me know when the update / correction has been added to your release and I'll confirm.

**Comment by thorsten** [comment]
Will be fixed with phpMyFAQ 2.9.0-alpha4
